core/src/main/scala/kafka/utils/Throttler.scala

```diff
@@ -27,32 +27,35 @@ import scala.math._
 
 /**
  * A class to measure and throttle the rate of some process. The throttler takes a desired rate-per-second
- * (the units of the process don't matter, it could be bytes or a count of some other thing), and will sleep for 
- * an appropriate amount of time when maybeThrottle() is called to attain the desired rate.
+ * (the units of the process don't matter, it could be bytes or a count of some other thing), and will sleep for
+ * an appropriate amount of time when {@link maybeThrottle(double)} is called to attain the desired rate.
  * 
  * @param desiredRatePerSec: The rate we want to hit in units/sec
  * @param checkIntervalMs: The interval at which to check our rate
  * @param throttleDown: Does throttling increase or decrease our rate?
+ * @param metricName: The metric name for this throttler
  * @param time: The time implementation to use
+ * @param units: The units of the {@code time}
  */
 @threadsafe
 class Throttler(@volatile var desiredRatePerSec: Double,
-                checkIntervalMs: Long = 100L,
+                checkIntervalMs: Long = 300L,
                 throttleDown: Boolean = true,
-                metricName: String = "throttler",
-                units: String = "entries",
+                metricName: String = "cleaner-io",
+                units: String = "bytes",
                 time: Time = Time.SYSTEM) extends Logging with KafkaMetricsGroup {
-  
+
   private val lock = new Object
   private val meter = newMeter(metricName, units, TimeUnit.SECONDS)
   private val checkIntervalNs = TimeUnit.MILLISECONDS.toNanos(checkIntervalMs)
-  private var periodStartNs: Long = time.nanoseconds
+  private var periodStartNs: Long = 0
```

Comment:
put default `periodStartNs` as 0, so that we can know if we need to initialize this value